variables.tf

```
@@ -39,16 +39,16 @@ variable "additional_ses_sendemail_account_ids" {
   default     = []
 }
 
-variable "aws_region" {
+variable "api_gateway_zone_id" {
   type        = string
-  description = "The AWS region to communicate with."
-  default     = "us-east-1"
+  description = "The Route 53 hosted zone ID for API Gateways."
```

Remove unnecessary capitalization:
```suggestion
  description = "The Route 53 hosted zone ID for API gateways."
```

---

Fixed in this commit [c14aa64](https://github.com/cisagov/cool-dns-cyber.dhs.gov/pull/92/commits/c14aa64dca18d92c60eec7aab6b358b685090546)

---

Thanks, though in the future, please create separate commits for the capitalization fix and the changes to the alias domain.